scripts/parachain/local.sh

```diff
@@ -91,13 +90,22 @@ start_collator() {
     --execution=wasm \
     --port=$relay_chain_port \
     --rpc-port=$relay_chain_rpc_port \
-    --tmp \
     --ws-port=$relay_chain_ws_port \
     -lcumulus_collator=trace \
-    -lruntime=trace
+    -lruntime=trace"
+    
+    $LAUNCH_PARACHAIN_CMD & node_pid=$!
+
+    sleep 30
+    DATA='{ "id":1, "jsonrpc":"2.0", "method":"author_insertKey", "params":["nmbs", "'"$seed"'", "'"$public_key"'"] }'
+    curl -H 'Content-Type: application/json' --data "$DATA" localhost:$collator_rpc_port
+
+    sleep 5
+    kill $node_pid
+    sleep 5
+    $LAUNCH_PARACHAIN_CMD
 }
 
 # Feel free to comment, add or remove collators. Just remember that #Validators > #Collators
 
-start_collator --alice 30333 9933 9944 32000 8200 9200 &> /dev/null & node_pid=$!
-start_collator --bob 30334 9934 9945 32001 8201 9201
\ No newline at end of file
+start_collator 30333 9933 9944 32000 8200 9200 "swap ketchup reopen speak diary estate jealous ostrich crater useless jaguar attack" "0x7225b9e36cc33da17d7f97053c7110559f21967afc09867345367b7de241d03c"
```

Comment:
* "swap ketchup reopen speak diary estate jealous ostrich crater useless jaguar attack" is the account id seed
* "0x7225b9e36cc33da17d7f97053c7110559f21967afc09867345367b7de241d03c" is the session key